### PR TITLE
Don't merge - componentName

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

File: content/creator/sdk7/architecture/custom-components.md [hunk 1]
```diff
@@ -68,26 +71,13 @@ const readOnlyInstance MyCustomComponent.deleteFrom(myEntity)
...
-```ts
-enum CustomComponentIds {
-  isButton = 2001,
-  isDoor = 2002,
-  pickedUp = 2003
-}
-```
+## About the componentName
 
+Each component must have a unique componentName, that differentiates it internally. You won't need to use this internal identifier anywhere else in your code. A good practice is to use the same name you assign to the component, but starting with a lower case letter, but all that really matters is that this identifier is unique within the project.
```
Comment (menduz) [inline content/creator/sdk7/architecture/custom-components.md]:
```suggestion
Each component must have a unique component name or identifier, that differentiates it internally. You won't need to use this internal identifier anywhere else in your code. A good practice is to use the same name you assign to the component, but starting with a lower case letter, but all that really matters is that this identifier is unique within the project.

<!-- UNCOMMENT WHEN EDITOR IS READY TO BE RELEASED
It is important to notice that this component name should never change, so pick it carefully. It is used to store the composites (a.k.a. prefabs) in a human readable format.
-->
```

File: content/creator/sdk7/interactivity/raycasting.md [hunk 2]
```diff
@@ -151,17 +151,17 @@ In some cases you may want to have several separate raycast queries running at t
 ```typescript
 // custom components
 const CubeOscilator = engine.defineComponent(
+  "cubeOscilator",
```
Comment (menduz) [inline content/creator/sdk7/interactivity/raycasting.md]:
```suggestion
  "CubeOscilator",
```

File: content/creator/sdk7/architecture/custom-components.md [hunk 3]
```diff
@@ -267,10 +257,10 @@ With your component defined and added to entities in your scene, you can create
 
 ```ts
 // define component
-export const WheelSpinComponent = engine.defineComponent({
+export const WheelSpinComponent = engine.defineComponent("wheelSpinComponent", {
```
Comment (menduz) [inline content/creator/sdk7/architecture/custom-components.md]:
```suggestion
export const WheelSpinComponent = engine.defineComponent("WheelSpinComponent", {
```

File: content/creator/sdk7/architecture/custom-components.md [hunk 4]
```diff
@@ -193,7 +183,7 @@ const MoveTransportData = {
   speed: Schemas.Float,
 }
 
-export const LerpTransformComponent = engine.defineComponent(MoveTransportData, COMPONENT_ID)
+export const LerpTransformComponent = engine.defineComponent("lerpTransformComponent", MoveTransportData)
```
Comment (menduz) [inline content/creator/sdk7/architecture/custom-components.md]:
```suggestion
export const LerpTransformComponent = engine.defineComponent("LerpTransformComponent", MoveTransportData)
```

File: content/creator/sdk7/architecture/custom-components.md [hunk 5]
```diff
@@ -146,7 +136,7 @@ const mySchema = {
 }
 
 //// component
-export const WheelSpinComponent = engine.defineComponent(mySchema, 2000)
+export const WheelSpinComponent = engine.defineComponent("wheelSpinComponent", mySchema)
```
Comment (menduz) [inline content/creator/sdk7/architecture/custom-components.md]:
```suggestion
export const WheelSpinComponent = engine.defineComponent("WheelSpinComponent", mySchema)
```

File: content/creator/sdk7/architecture/custom-components.md [hunk 6]
```diff
@@ -132,10 +122,10 @@ You can chose to create the schema inline while defining the component, or for m
 
 ```ts
 // Option 1: Inline definition
-export const WheelSpinComponent = engine.defineComponent({
+export const WheelSpinComponent = engine.defineComponent("wheelSpinComponent", {
```
Comment (menduz) [inline content/creator/sdk7/architecture/custom-components.md]:
```suggestion
export const WheelSpinComponent = engine.defineComponent("WheelSpinComponent", {
```

File: content/creator/sdk7/architecture/custom-components.md [hunk 7]
```diff
@@ -119,10 +109,10 @@ A schema describes the structure of the data inside a component. A component can
 Every field in the schema must include a type declaration. You can only use the special schema types provided by the SDK. For example, use the type `Schemas.Boolean` instead of type `boolean`. Write `Schemas.` and your IDE will display all the available options.
 
 ```ts
-export const WheelSpinComponent = engine.defineComponent({
+export const WheelSpinComponent = engine.defineComponent("wheelSpinComponent", {
```
Comment (menduz) [inline content/creator/sdk7/architecture/custom-components.md]:
```suggestion
export const WheelSpinComponent = engine.defineComponent("WheelSpinComponent", {
```

File: content/creator/sdk7/3d-essentials/move-entities.md [hunk 8]
```diff
@@ -463,7 +453,7 @@ const PathTransportData = {
   pathTargetIndex: Schemas.Int
 }
 
-export const LerpTransformComponent = engine.defineComponent(PathTransportData, COMPONENT_ID)
+export const LerpTransformComponent = engine.defineComponent("lerpTransformComponent", PathTransportData)
```
Comment (menduz) [inline content/creator/sdk7/3d-essentials/move-entities.md]:
```suggestion
export const LerpTransformComponent = engine.defineComponent("LerpTransformComponent", PathTransportData)
```

File: content/creator/sdk7/3d-essentials/move-entities.md [hunk 9]
```diff
@@ -392,16 +386,14 @@ Often these non-linear transitions can breathe a lot of life into a scene. A mov
...
-
 const MoveTransportData = {
   start: Schemas.Vector3,
   end: Schemas.Vector3,
   fraction: Schemas.Float,
   speed: Schemas.Float,
 }
 
-export const LerpTransformComponent = engine.defineComponent(MoveTransportData, COMPONENT_ID)
+export const LerpTransformComponent = engine.defineComponent("lerpTransformComponent", MoveTransportData)
```
Comment (menduz) [inline content/creator/sdk7/3d-essentials/move-entities.md]:
```suggestion
export const LerpTransformComponent = engine.defineComponent("LerpTransformComponent", MoveTransportData)
```

File: content/creator/sdk7/3d-essentials/move-entities.md [hunk 10]
```diff
@@ -335,16 +331,14 @@ To implement this lerp in your scene, we recommend creating a custom component t
...
-
 const ScaleTransportData = {
   start: Schemas.Number,
   end: Schemas.Number,
   fraction: Schemas.Float,
   speed: Schemas.Float,
 }
 
-export const ScaleTransformComponent = engine.defineComponent(ScaleTransportData, COMPONENT_ID)
+export const ScaleTransformComponent = engine.defineComponent("scaleTransformComponent", ScaleTransportData)
```
Comment (menduz) [inline content/creator/sdk7/3d-essentials/move-entities.md]:
```suggestion
export const ScaleTransformComponent = engine.defineComponent("ScaleTransformComponent", ScaleTransportData)
```

File: content/creator/sdk7/3d-essentials/move-entities.md [hunk 11]
```diff
@@ -163,16 +163,14 @@ To implement this `lerp()` in your scene, we recommend creating a [custom compon
...
-
 const MoveTransportData = {
   start: Schemas.Vector3,
   end: Schemas.Vector3,
   fraction: Schemas.Float,
   speed: Schemas.Float,
 }
 
-export const LerpTransformComponent = engine.defineComponent(MoveTransportData, COMPONENT_ID)
+export const LerpTransformComponent = engine.defineComponent("lerpTransformComponent", MoveTransportData)
```
Comment (menduz) [inline content/creator/sdk7/3d-essentials/move-entities.md]:
```suggestion
export const LerpTransformComponent = engine.defineComponent("LerpTransformComponent", MoveTransportData)
```

**Comment by nearnshaw** [comment]
Merged into branch with full 7.1 release https://github.com/decentraland/documentation/pull/103

**Pull request closed without merging**